openmetadata-ui/src/main/resources/ui/src/axiosAPIs/miscAPI.ts

```diff
@@ -209,15 +209,17 @@ export const getSearchedTeams = (
   return searchData(queryString, from, size, '', '', '', SearchIndex.TEAM);
 };
 
-export const getSearchedUsersAndTeams = (
+export const getSearchedUsersAndTeams = async (
   queryString: string,
   from: number,
   size = 10
 ) => {
-  return searchData(queryString, from, size, '', '', '', [
+  const response = await searchData(queryString, from, size, '', '', '', [
     SearchIndex.USER,
     SearchIndex.TEAM,
   ]);
+
+  return response.data;
```

Comment:
can we wrap this in try/catch?

Reply:
No It's inside API file and we need error at caller side, If we wrap try/catch here we don't get error at caller side

Reply:
oh ok, thanks I did not notice the file name!